README.md

```diff
@@ -45,6 +45,14 @@ NumPy requires `pytest` and `hypothesis`.  Tests can then be run after installat
 
     python -c 'import numpy; numpy.test()'
 
+Code of Conduct
+----------------------
+
+NumPy is a community-driven open source project developed by a diverse group of
+[contributors](https://numpy.org/gallery/team.html). The NumPy leadership has made a strong
```

Comment:
This link is about to change to https://numpy.org/teams/ when we merge https://github.com/numpy/numpy.org/pull/510. So let's do that first, and then update this PR?

Reply:
See #20740.